vMenu/data/VehicleData.cs

```diff
@@ -38,6 +38,71 @@ public VehicleColor(int id, string label)
                         CitizenFX.Core.Native.API.AddTextEntry("VERY_DARK_BLUE", "Very Dark Blue");
                     }
                 }
+                // Chameleon Colour Labels //
```

Comment:
The code checker is finding an issue here due to how big the list it.
I am not sure if the AddTextEntry's are needed for the Chameleon Colours. Added them just in case.